temporalio/lib/temporalio/worker.rb

```diff
@@ -52,7 +53,9 @@ class Worker
       :workflow_payload_codec_thread_pool,
       :unsafe_workflow_io_enabled,
       :deployment_options,
-      :debug_mode
+      :debug_mode,
+      :workflow_task_poller_behavior,
+      :activity_task_poller_behavior
```

Comment:
```suggestion
      :workflow_task_poller_behavior,
      :activity_task_poller_behavior,
      :debug_mode
```

Might as well keep the same order as `initialize` (and update where you instantiate the options too)